### PR TITLE
update Readme.md correct usage of date function

### PR DESCRIPTION
**Pull request by JulianoEngineer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,12 +72,11 @@ And the corresponding "date" function that returns a date for
 each data point.
 
 ```js
-var eventDropsChart = d3.chart.eventDrops();
+var eventDropsChart = d3.chart.eventDrops()
+                              .date(d => d.date);
```
Comment (jpetitcolas) [inline README.md]: s/                              /    /

Comment (jpetitcolas) [inline README.md]: Can you fix indentation? 

I tried to tell @sedy-bot to do it, but it seems it is not in a correct mood today (ping @Kmaschta). :p

**Comment by jpetitcolas** [comment]
Nice catch! :)

File: README.md (hunk 1 above)
Comment (JulianoEngineer) [inline README.md]: Where is the problem with the indentation?

Comment (Kmaschta) [inline README.md]: @jpetitcolas If you want to replace a space by a space, I would react just as @sedy-bot!

Comment (jpetitcolas) [inline README.md]: @JulianoEngineer : I would simply add 4 spaces. It is too far away here. :)

```
var eventDropsChart = d3.chart.eventDrops()
    .date(d => d.date);
```

@Kmaschta: my bad. Seems that GitHub replaced several spaces by a single one. @sedy-bot, I beg your pardon!

**Comment by jpetitcolas** [comment]
Thanks! :)

**Pull request merged**